google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2154,6 +2179,82 @@ public ApiFuture<Empty> asyncClose() {
     }
   }
 
+  private final class MultiplexedSessionWaiterFuture
+      extends ForwardingListenableFuture<MultiplexedSession> {
+
+    private final Duration MAX_MULTIPLEXED_SESSION_WAIT_TIMEOUT = Duration.ofSeconds(60);
```

Comment:
I still think that we should remove most (all?) of this, because almost all of the code here is not used anyways, unless the user has changed some default settings. See my further comments below.

Reply:
Done.